codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexTransportTest.java

```diff
@@ -435,14 +442,16 @@ public void operationComplete(Future<Channel> future) {
         }
     }
 
-    @Test(timeout = 5000L)
+    @Test
+    @Timeout(value = 5000L, unit = MILLISECONDS)
     public void testFireChannelReadAfterHandshakeSuccess_JDK() throws Exception {
         assumeTrue(SslProvider.isAlpnSupported(SslProvider.JDK));
         testFireChannelReadAfterHandshakeSuccess(SslProvider.JDK);
     }
 
     @Disabled("This fails atm... needs investigation")
-    @Test(timeout = 5000L)
+    @Test
+    @Timeout(value = 5000L, unit = MILLISECONDS)
```

Comment:
The `@Timeout` annotation can also be applied at the class-level, and would then apply to all tests in the class, individually.

Reply:
I think we can do this as a follow as I just wanted to port over what we had...